CMakeLists.txt

```diff
@@ -6,6 +6,8 @@ SET(PROJECT_NAME crocoddyl)
 SET(PROJECT_DESCRIPTION "Contact RObot COntrol by Differential DYnamic programming Library (Crocoddyl)")
 SET(PROJECT_URL https://github.com/${PROJECT_NAMESPACE}/${PROJECT_NAME})
 
+OPTION(INSTALL_DOCUMENTATION "Generate and install the documentation" OFF)
```

Comment:
@wxmerkt -- Focal/clang/Debug job is failing, and this seems to be the cause. Should I enable this flag here https://github.com/loco-3d/crocoddyl/blob/master/.github/workflows/industrial_ci_action.yml#L29?

Reply:
I don't think that is the case. The issue is likely that unittests with Debug mode just take too long to run and never output to the console. The new finite difference might allocate (I didnt check the source code but that would be my first hunch given the symptoms) or just be slow in Debug mode

Reply:
Yes, it is much slower now as we need to do much more computations. I don't think we are allocating extra data.

I need to think about what strategy to take. Options are: 
  - deactivating the computation of Hessians in Debug mode, or
  - reducing the number of tests in Debug mode.

In any case, we are not checking the cost Hessians in the unit tests as the numerical error associated with the Gauss-Newton approximation is not possible to account for. This is a kind of pity, as I would like to check these Hessians as well.

Do you have any opinion about the difference options?

Reply:
We may need the former option for offline development but could guard it with a `#ifndef` to deactivate during unittesting since it's not checked? So `#ifndef SKIP_FINITE_DIFFERENCE_HESSIANS` or similar around that computation and we then define that for CI Debug builds?

Reply:
Good point! Thanks for your feedback :)